src/components/backend-ai-splash.ts

```diff
@@ -7,7 +7,7 @@ import {css, CSSResultGroup, html, LitElement} from 'lit';
 import {customElement, property} from 'lit/decorators.js';
 
 import 'weightless/button';
-import 'weightless/dialog';
+import {Dialog} from 'weightless/dialog';
```

Comment:
`wl-dialog` does not have show / hide method. how does it work?

Reply:
Probably, This seems to be another example of legacy.

Reply:
I found the code inheriting `OverlayBehavior` in `wl-dialog`. OverlayBehavior has internal methods that control dialog.

ref: https://github.com/andreasbm/weightless/blob/master/src/lib/behavior/overlay/overlay-behavior.ts#L190